packages/progress/src/index.vue

```diff
@@ -17,7 +17,10 @@
     <div v-if="type === 'line'" class="el-progress-bar">
       <div class="el-progress-bar__outer" :style="{height: `${strokeWidth}px`}">
         <div class="el-progress-bar__inner" :style="barStyle">
-          <div v-if="showText && textInside" class="el-progress-bar__innerText">{{ content }}</div>
+          <div v-if="(showText || $slots.default) && textInside" class="el-progress-bar__innerText">
+            <slot v-if="$slots.default" v-bind="slotData"></slot>
```

Comment:
Simply use this expression
```suggestion
<slot v-bind="slotData">
  <span>{{ content }}</span>
</slot>
```